### PR TITLE
change cfu on github actions

### PR DESCRIPTION
**Pull request by gorouflex**
This will only check the `main.py` file, not the whole repository. Therefore, any changes outside the `main.py` file will not affect this CFU feature (#16)

### CODE REVIEW

**Comment by gorouflex** [comment]
i'm also add a feature that if this script run on GitHub Actions, after complete the process than will delete the Config folder ( i don't know if it's useful or not? )

File: main.py [hunk 1]
```diff
@@ -420,4 +421,9 @@ def main() -> None:
 
 if __name__ == '__main__':
     main()
+    if os.getenv('GITHUB_ACTIONS') == 'true':
+       try:
+         shutil.rmtree(config_folder)
+       except:
```
Comment (MrLoLf) [inline main.py:427]: It would be great to see, which exception you are trying to catch. Please add this in to the except and I'll be happy to merge this.

**Review by MrLoLf — CHANGES_REQUESTED** [review]
It would be great to see, which exception you are trying to catch. Please add this in to the except and I'll be happy to merge this.

**Comment by gorouflex** [comment]
@MrLoLf Can you explain more? I still don't know what do you mean

**Comment by MrLoLf** [comment]
> @MrLoLf Can you explain more? I still don't know what do you mean

Yeah, no problem. It is very common to specify which error you are trying to catch with the except. So people that read this code understand, that this error could occur and when some other error was thrown in the except it wouldn't be handled by accident. So it's easier to spot bugs. When done with the correct error handling, it should like this. 
```python
try:
        urls_conf: dict[str, str] = {'login': cfg.get('Url', 'login'),
                                     'pot': cfg.get('Url', 'pot'),
                                     'balance': cfg.get('Url', 'balance'),
                                     'achievements': cfg.get('Url', 'achievements'),
                                     'achievement_claim': cfg.get('Url', 'achievement_claim')}
    except configparser.NoOptionError:
        # something to do on the exception here
```
Note the specified Error on the except.

**Comment by gorouflex** [comment]
Ok i’ll do it later

**Comment by gorouflex** [comment]
@MrLoLf Done

**Pull request merged**